pooch/tests/test_downloaders.py

```diff
@@ -39,6 +48,18 @@ def test_ftp_downloader():
         assert os.path.exists(outfile)
 
 
+@pytest.mark.skipif(SFTPDownloader is None, reason="requires paramiko")
```

Comment:
If implementing the changes mentioned above, this can me `skipif(paramiko is None, ...` instead.